Modix.Services/Starboard/StarboardHandler.cs

```diff
@@ -15,12 +16,14 @@ public class StarboardHandler :
     {
         private readonly StarboardService _service;
         private readonly IDesignatedChannelService _designatedChannelService;
-        private const string _contentString = "**{0}** {1} ID: {2}";
+        private readonly IQuoteService _quoteService;
+        private const string _contentString = "**{0}** {1}";
```

Comment:
An alternative might be to turn this into a private method that uses the more modern `$""`.